### DISCUSSION TITLE
List number of queued jobs in GitHub-hosted runners view

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#12077] [Actions] [answered]
The new runners view shows:
<img width="945" alt="image" src="https://user-images.githubusercontent.com/2119212/155657503-bd68d341-a793-4ccf-a456-5fbbc6e53d62.png">

It doesn't say how many queued jobs there are / link to them. / Conversely, it doesn't tell me that there are 0 jobs waiting.

**Comment by ethomson** [comment, marked as answer]
Thanks as always, @jsoref - we are looking at how we visualize runners and utilization and I will pass this along to the team so that they have this feedback as they're planning the next iteration.

**Reply by jsoref** [reply]
I'd also like to have access to this screen for my user account, not just my orgs accounts.
